.github/ISSUE_TEMPLATE/oss_contribution.yml

```diff
@@ -0,0 +1,57 @@
+name: OSS Contribution
```

Comment:
```suggestion
name: Data Contribution
```
What about modifying the name to "Data Contribution"?

Reply:
Done